pygsti/report/reportables.py

```diff
@@ -2218,9 +2259,7 @@ def predicted_rb_number(model_a, model_b):
     -------
     float
     """
-    #TEMPORARILY disabled b/c RB analysis is broken
-    #from ..extras.rb import theory as _rbtheory
-    return -1.0  # _rbtheory.predicted_rb_number(model_a, model_b)
+    return _tools.rbtheory.predicted_rb_number(model_a, model_b)
```

Comment:
Ran this change past @tjproct and also asked him to take a look at the rest of the code for the predicted RB number table's construction and no immediate concerns were raised.